src/components/books/Books.js

```diff
@@ -4,9 +4,11 @@ import { fetchBooks, removeAllBooks } from '../../redux/books/books';
 import Book from '../book/Book';
 import AddNewBook from '../form/AddNewBook';
 
+// Books Component
 function Books() {
   const dispatch = useDispatch();
 
+  // fetch books from the store
   useEffect(() => {
     dispatch(fetchBooks());
     return () => {
```

Comment:
The app does show a blank page while running it locally. It does not show any content on my system. Please remove the dispatcher which removes all books from the store in `useEffect` hook as this may be the problem.